.gitignore

```diff
@@ -9,3 +9,5 @@ logs/
 *.files
 *.includes
 CMakeLists.txt.user*
+
+.DS_Store
```

Comment:
This should be in your global gitignore but since not everyone has that it's probably a good idea anyway to add it.